_docs/setup/kubernetes/sidecar-injection.md

```diff
@@ -148,7 +148,7 @@ secret for the sidecar injector webhook to consume.
 
 _Note_: Kubernetes CA approval requires permissions to create and approve CSR. See 
 [https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster ](https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/) and 
-[install/kubernetes/webhook-create-signed-cert.sh](https://raw.githubusercontent.com/istio/istio/master/install/kubernetes/webhook-create-signed-cert.sh) for more information.
+[install/kubernetes/webhook-create-signed-cert.sh](https://raw.githubusercontent.com/istio/istio/release-0.7/install/kubernetes/webhook-create-signed-cert.sh) for more information.
```

Comment:
This file should be in the release tgz

Reply:
curl on that link works - and it matches the existing intent of the document.  What I didn't want to do was get into a document rewrite to unblock the gate. :)

Not sure if github does automatic tar/gz removal - I'll give it a try and push a new PR if you think it is necessary.

Cheers
-steve